### ISSUE TITLE
Astap error (exception during solving)

### ISSUE THREAD

**Issue by Vandewattyne01** [mworion/MountWizzard4#338]
In astap i'm able to solve the image but not in MW4.
It shows the message Astap error (exception during solving) and in the logs I see the msg:

 plateSolve.py][ 286] Start thread for solving
[2023-12-02 17:52:43.763][U][     mainApp.py][ 371] Message window: [Image - Solving - M31.fits]
[2023-12-02 17:52:43.766][C][       astap.py][ 119] error: [WinError 5] Access is denied happened
[2023-12-02 17:52:43.766][W][       astap.py][ 181] ASTAP error [Exception during solving] in [C:\Astro\MW4\image\M31.fits]
[2023-12-02 17:52:43.767][U][     mainApp.py][ 371] Message window: [Image - Solving error - ASTAP error: [Exception during solving]]
[2023-12-02 17:52:43.769][D][  plateSolve.py][ 253] Finished clear thread for solving

**Comment by mworion** [comment]
Hi,
if you don't mit, please post the image you want to solve here (zip it before and drop it here) and please raise the log level for MW4 to debug (to see a little bit more when running)
Michel

**Comment by Vandewattyne01** [comment]
Hello Michael,
This is the file. In the software astap i can solve it. 
I installed MW4 on another PC with the same result.

[m31.zip](https://github.com/mworion/MountWizzard4/files/13588226/m31.zip)

**Comment by mworion** [comment]
Hi 
some hints:

- the RA coordinate if a little off: either you start modeling or your mount system time / orientation / model is wrong or not correctly set. DEC is OK. 

- default for solving is a search radius of 10 for ASTAP. This is done to get fast results in case of having a reasonable model and good parameters. If you increase the search radius to 180 degree (all sky solving), ASTAP will plate solve this file within MW4. To be honest: ASTAP natively falls back to all sky in that case as well.

- but with a reasonable setup you should not need using all sky setup. Please check some parameters of your setup: the scale in the header is calculated (from SGPro) to 286 (arcsec/pixel) which leads to an extreme wide field of view. ASTAP recognizes that and outputs:

```
ASTAP took 0.2851221561431885s return code: 2 [/Users/mw/Desktop/M31_1.fits] stderr:  stdout: Could not find w08 star database. Will try with an other database. Using star database D50 Very large FOV, use W08 database!  Cropping image x 0.03 █ █ █ █ █ █ Warning, remaining image dimensions too low! Try to REDUCE OR REMOVE DOWNSAMPLING. Set this option in stack menu, tab alignment. Only 1 stars found in image. Abort No solution found!  :( Warning, reached the limit of the star database! 
```
- ASTAP corrects this by doing all sky and being able to solve it anyway. If you get the solve, you see that the reals scale is about 2.86 arcs / pixel which is a quite normal number. If you change this setting in the FITS header of the image file, MW4 together with ASTAP with default settings solves the image within 1 second. 

- Why does SGPro enters a wrong parameter - I do not know, because the pixel size with binning equals to 7.52 (quite reasonable) and with the FL of 500mm the scale would be  206 * (pixel size) / (focal length) = 206 * 7.52 / 500 = 3.09. So from my point SGPro does a mistake by entering a wrong number

Hope this helps.
Michel

**Comment by Vandewattyne01** [comment]
Hi Michel,
In SGP i did the correction of the scale and the PA is also better now. But still I get the same error. Also I used some older subs and its the same. Is it working on the MW4 on you PC to solve it?

**Comment by mworion** [comment]
Hi,
yes if you set ASTAP config in MW4 (search radius to 180 deg) to all sky, ASTAP solves the images. I'm think of making this the default as it should work in any cases.
Michel

**Comment by mworion** [comment]
But anyhow I published a 3.2.5, which should be better.
